### ISSUE TITLE
Can't run local testnet: decoding bech32 failed: invalid index of 1

### ISSUE THREAD

**Issue by NodeGuy** [cosmos/voyager#1507]
UI Version: 0.8001.1

Description:

```
$ yarn build:gaia
$ yarn build:local --overwrite=true
$ ./builds/Gaia/darwin_amd64/gaiad start --home ~/.gaiad-testnet
$ yarn start local-testnet
```

The console shows:

<img width="804" alt="screen shot 2018-10-31 at 7 25 04 pm" src="https://user-images.githubusercontent.com/2148537/47824334-ba6bb100-dd42-11e8-9a0f-01567ce8c415.png">

Failed network responses:

`decoding bech32 failed: invalid index of 1`
`invalid bech32 prefix. Expected cosmosaccaddr, Got cosmos`

**Comment by faboweb** [comment]
As you can see in the error log, Voyager is connecting to the remove full node. To connect to your local node you have to specify the endpoints:
`LCD_URL=http://localhost:9070 RPC_URL=http://localhost:26657 yarn start local-testnet`

**Comment by faboweb** [comment]
Fixed in https://github.com/cosmos/voyager/pull/1508

**Comment by fedekunze** [comment]
Closed via #1508
